src/cogs/admin_cog.py

```
@@ -39,16 +39,111 @@ async def ping(self, interaction: discord.Interaction):
+                    "> *Example:* `/github_tree owner/repo 3`\n"
+                    "> Supports GitHub URLs too!"
+                ),
+                inline=False
+            )
+
+            embed.set_footer(text="Made with Discord.py | Powered by GitHub API")
+            embed.set_thumbnail(url=self.bot.user.avatar.url if self.bot.user.avatar else None)
+
+            await interaction.followup.send(embed=embed)
+
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)